plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/wizard/deployimage/DeployImagePage.java

```diff
@@ -469,4 +478,33 @@ protected IStatus updateUI(IProgressMonitor monitor) {
 		}
 	}
 	
+    private void onNewDockerConnectionClicked() {
+        try {
+            // run in job to enforce busy cursor which doesnt work otherwise
+            WizardUtils.runInWizard(new UIUpdatingJob("Opening new Docker connection wizard...") {
+
+                @Override
+                protected IStatus run(IProgressMonitor monitor) {
+                    return Status.OK_STATUS;
+                }
+
+                @Override
+                protected IStatus updateUI(IProgressMonitor monitor) {
+                    NewDockerConnection newDockerConnectionWizard = new NewDockerConnection();
+                    int result = new OkCancelButtonWizardDialog(getShell(), newDockerConnectionWizard).open();
+                    // set docker connection to reflect changes that happened in
+                    // docker connection wizard
+                    if (Dialog.OK == result) {
+                        IDockerConnection dockerConnection = newDockerConnectionWizard.getDockerConnection();
+                        if (dockerConnection != null) {
+                            model.setDockerConnection(dockerConnection);
+                        }
+                    }
```

Comment:
Will new docker connection appear in the list of combo? Method setDockerConnection does not fire a change event for property "dockerConnections".